plugins/extrusions/extrusions.py

```diff
@@ -0,0 +1,146 @@
+from typing import TypeVar, Union, Tuple
+
+# import importlib.resources
+import pkg_resources
+import cadquery as cq
+from cadquery.occ_impl import importers
+from cadquery.occ_impl.geom import Vector, Location
+
+T = TypeVar("T", bound="Workplane")
+
+
+def _extrusion(
+    size: Tuple[float, float],
+    profile,
+    length: float,
+    centered: Union[bool, Tuple[bool, bool, bool]],
+):
+    """
+    Return an extrusion with the specified profile and length
+
+    :param size: size of the extrusion, used for centering
+    :param profile: filename of the extrusion's profile
+    :param length: length of the extrusion
+    :type length: float > 0
+    :param centered: If True, the extrusion will be centered around the reference point.
+        If False, the corner of the extrusion will be on the reference point and it will
+        extend in the positive x, y and z directions. Can also use a 3-tuple to
+        specify centering along each axis.
+    """
+    # file = (
```

Comment:
Is this commented code from an older evolution of the profile loading implementation?

Reply:
No, the opposite actually. `pkg_resources` is being deprecated in favour of `importlib.resources`, which is available in Python 3.9. I originally used this, but it failed to build, so I guess the build environment is on an earlier version of Python. The commented out code can be uncommented once the switch the Python 3.9 is complete.